README.md

```diff
@@ -94,7 +109,9 @@ See `config.rb.sample` for more information.
 ## Cluster Setup
 
 Launching a CoreOS cluster on Vagrant is as simple as configuring `$num_instances` in a `config.rb` file to 3 (or more!) and running `vagrant up`.
```

Comment:
I know this isn't related to your change, but this should be Container Linux... and possibly in other places. Happy to do in a separate PR if you don't want to.